### PR TITLE
[trace-view] Added support for continue

### PR DESCRIPTION
**Pull request by awelc**
## Description 

This PR adds support for "continue" debugger action. It also makes implementation of "next" ("step over") and "step out" actions faithful to their intended semantics (i.e., executing all instruction in a function stepped over or stepped out of). These changes do not actually change the current behavior of the viewer - they are done in preparation for implementing variable value tracking and displaying.

This PR also includes a refinement of the "step out" action implementation. As also explained in the code comment, previously in the following line of code, after entering `foo` and stepping out of it, we would immediately step into `bar`, which could be quite confusing for the user:
```
assert(foo() == bar());
```

Finally, this PR also includes some formatting changes for lines that were a bit too long

## Test plan 

Tested manually

### CODE REVIEW

File: external-crates/move/crates/move-analyzer/trace-adapter/src/runtime.ts [hunk 1]
```diff
@@ -198,45 +201,63 @@ export class Runtime extends EventEmitter {
...
+                this.frameStack.frames.pop();
+                if (currentEvent.id === currentFrameID) {
+                    break;
+                }
             }
         }
-        this.frameStack.frames.pop();
-        return this.step(false);
+
+        // Do not skip to same line when stepping out as this may lead
```
Comment (awelc) [inline external-crates/move/crates/move-analyzer/trace-adapter/src/runtime.ts:221]: This change is unrelated to the "continue" action implementation, but it's small enough that I decided to sneak it in as part of this PR....

File: external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts [hunk 2]
```diff
@@ -148,7 +149,10 @@ export class MoveDebugSession extends LoggingDebugSession {
         this.sendEvent(new InitializedEvent());
     }
 
-    protected async launchRequest(response: DebugProtocol.LaunchResponse, args: ILaunchRequestArguments): Promise<void> {
+    protected async launchRequest(
```
Comment (awelc) [inline external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts:152]: This (and the following) formatting changes were not necessary to achieve the stated goal of this PR but will make the code look better (as a side-note, I don't like TS auto-formatter not formatting these automatically)

**Review by rvantonder — APPROVED** [review]
looks good!
